package.json

```diff
@@ -40,6 +40,10 @@
 		"docker:ngrok": "yarn docker:compose -f docker/docker-compose-ngrok.yml",
 		"docker:ngrok-up": "yarn docker:ngrok up",
 		"docker:ngrok-down": "yarn docker:ngrok down",
+		"docker:jt": "echo 'The tool is only available to Automatticians. Please visit https://github.com/Automattic/jurassic-tube for installation instructions.'",
```

Comment:
I think I would still remove the GitHub link, and instead have a shorthand link to a P2 post or a FG page, to avoid people digging too much into this.